src/main/blackbox/blackbox.c

```diff
@@ -1191,6 +1207,8 @@ static void loadMainState(timeUs_t currentTimeUs)
         blackboxCurrent->gyroUnfilt[i] = lrintf(gyro.gyroADC[i] * blackboxHighResolutionScale);
 #if defined(USE_ACC)
         blackboxCurrent->accADC[i] = lrintf(acc.accADC.v[i]);
+        STATIC_ASSERT(offsetof(quaternion_t, w), "Code expects quaternion in w, x, y, z order");
```

Comment:
```suggestion
        STATIC_ASSERT(offsetof(quaternion_t, w) != 1, "Code expects quaternion in w, x, y, z order");
```

Reply:
yes, i've read - the bool only, there are no default data conversion